ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/IDisplayable.java

```diff
@@ -0,0 +1,39 @@
+package com.github.migi_1.Context.model.entity;
+
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Spatial;
+
+/**
+ * Interface for all items that are displayed.
+ *
+ * @author Marcel
+ *
+ */
+public interface IDisplayable {
+
+    /**
+     * Get the physical model of the object.
+     * @return The physical model.
+     */
+    Spatial getModel();
+
+    /**
+     * Set the physical model of the object.
```

Comment:
Sets*

In Javadoc, please refer to methods in third person instead of second :)